scripts/upgradeable/transpile.sh

```
@@ -1,47 +1,19 @@
-  -x 'contracts-exposed/**/*' \
-  -x 'contracts/mocks/**/*Proxy*.sol' \
-  -x 'contracts/proxy/**/*Proxy*.sol' \
-  -x 'contracts/proxy/beacon/UpgradeableBeacon.sol' \
-  -p 'contracts/access/manager/AccessManager.sol' \
-  -p 'contracts/finance/VestingWallet.sol' \
-  -p 'contracts/governance/TimelockController.sol' \
-  -p 'contracts/metatx/ERC2771Forwarder.sol' \
-  -n \
-  -N 'contracts/mocks/**/*' \
-  -q '@openzeppelin/'
+npx hardhat transpile --settings $DIRNAME/transpile.config.json
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)